README.md

```diff
@@ -33,7 +33,7 @@ learning frameworks.
 - Configuration file examples for each supported compression algorithm.
 - Git patches for prominent third-party repositories ([huggingface-transformers](https://github.com/huggingface/transformers)) demonstrating the process of integrating NNCF into custom training pipelines
 - Exporting PyTorch compressed models to ONNX\* checkpoints and TensorFlow compressed models to SavedModel or Frozen Graph format, ready to use with [OpenVINO&trade; toolkit](https://github.com/openvinotoolkit/).
-- Support for compression-aware model training via the [adaptive-compression training loop](./docs/Usage.md#accuracy-aware-model-training).
+- Support for accuracy-aware model training pipelines via the [adaptive compression level training loop](./docs/Usage.md#accuracy-aware-model-training) and [early exit training loop](./docs/Usage.md#accuracy-aware-model-training).
```

Comment:
Is it ok that you have links to the same tag?

Reply:
Updated